src/lib/components/workers-list.svelte

```diff
@@ -5,57 +5,66 @@
   import { formatDate } from '$lib/utilities/format-date';
 
   import EmptyState from '$lib/holocene/empty-state.svelte';
+  import Table from '$lib/holocene/table/table.svelte';
+  import TableHeaderRow from '$lib/holocene/table/table-header-row.svelte';
   import type { GetPollersResponse } from '$lib/services/pollers-service';
 
   export let taskQueue: string;
   export let workers: GetPollersResponse;
 </script>
 
 <section class="flex flex-col gap-4">
-  <h3 class="text-lg font-medium">
+  <h2 class="text-lg font-medium">
     Task Queue: <span class="select-all font-normal">{taskQueue}</span>
-  </h3>
-  <section
-    class="flex w-full flex-col rounded-lg border-[3px] border-gray-900 bg-white"
-  >
-    <div class="flex flex-row bg-gray-900 p-2 text-white">
-      <div class="w-6/12 text-left">ID</div>
-      <div class="w-2/12 text-left">Last Accessed</div>
-      <div class="w-2/12 text-center">Workflow Task Handler</div>
-      <div class="w-2/12 text-center">Activity Handler</div>
-    </div>
+  </h2>
+  <Table class="mb-6 w-full min-w-[600px] table-fixed">
+    <TableHeaderRow slot="headers">
+      <th class="table-cell w-6/12">ID</th>
+      <th class="table-cell w-2/12">Last Accessed</th>
+      <th class="table-cell w-2/12">
+        <p class="text-center">Workflow Task Handler</p>
+      </th>
+      <th class="table-cell w-2/12 text-center">
+        <p class="text-center">Activity Handler</p>
+      </th>
+    </TableHeaderRow>
     {#each workers?.pollers as poller (poller.identity)}
-      <article
-        class="flex h-full w-full flex-row border-b-2 p-2 no-underline last:border-b-0"
-        data-cy="worker-row"
-      >
-        <div class="links w-6/12 text-left" data-cy="worker-identity">
+      <tr class="table-row w-full" data-cy="worker-row">
+        <td />
+        <td class="text-left" data-cy="worker-identity">
           <p class="select-all">{poller.identity}</p>
-        </div>
-        <div class="links w-2/12 text-left" data-cy="worker-last-access-time">
-          <h3>
-            <p class="select-all">
-              {formatDate(poller.lastAccessTime, $timeFormat)}
-            </p>
-          </h3>
-        </div>
-        <div class="flex w-2/12 justify-center" data-cy="workflow-poller">
+        </td>
+        <td class="text-left" data-cy="worker-last-access-time">
+          <p class="select-all">
+            {formatDate(poller.lastAccessTime, $timeFormat)}
+          </p>
+        </td>
+        <td data-cy="workflow-poller">
           {#if poller.taskQueueTypes.includes('WORKFLOW')}
-            <Icon name="checkmark" class="text-blue-700" />
+            <Icon
+              name="checkmark"
+              class="m-auto text-blue-700"
+              title="exists"
+            />
           {:else}
-            <Icon name="close" class="text-primary" />
+            <Icon name="close" class="m-auto text-primary" title="none" />
```

Comment:
Are there better/more descriptive titles we could use here besides `none` and `exists`?

Reply:
**Update:** I've just gone with `yes` and `no`.